kraken/pvc/pvc_scenario.py

```diff
@@ -46,64 +46,50 @@ def run(scenarios_list, config):
                 if pvc_name:
                     if pod_name:
                         logging.info(
-                            "pod_name '%s' will be overridden from the pod mounted in the PVC" % (str(pod_name))
+                            "pod_name '%s' will be overridden with one of the pods mounted in the PVC" % (str(pod_name))
                         )
-                    command = "kubectl describe pvc %s -n %s | grep -E 'Mounted By:|Used By:' | grep -Eo '[^: ]*$'" % (
-                        str(pvc_name),
-                        str(namespace),
-                    )
-                    logging.debug("Get pod name command:\n %s" % command)
-                    pod_name = runcommand.invoke(command, 60).rstrip()
-                    logging.info("Pod name: %s" % pod_name)
-                    if pod_name == "<none>":
+                    pvc = kubecli.get_pvc_info(pvc_name, namespace)
+                    try:
+                        pod_name = random.choice(pvc.podNames)
+                        logging.info("Pod name: %s" % pod_name)
+                    except Exception:
                         logging.error(
                             "Pod associated with %s PVC, on namespace %s, not found" % (str(pvc_name), str(namespace))
                         )
                         sys.exit(1)
 
                 # Get volume name
-                command = 'kubectl get pods %s -n %s -o json | jq -r ".spec.volumes"' % (
-                    str(pod_name),
-                    str(namespace),
-                )
-                logging.debug("Get mount path command:\n %s" % command)
-                volumes_list = runcommand.invoke(command, 60).rstrip()
-                volumes_list_json = json.loads(volumes_list)
-                for entry in volumes_list_json:
-                    if len(entry["persistentVolumeClaim"]["claimName"]) > 0:
-                        volume_name = entry["name"]
-                        pvc_name = entry["persistentVolumeClaim"]["claimName"]
+                pod = kubecli.get_pod_info(name=pod_name, namespace=namespace)
+                
+                if pod is None:
+                    sys.exit(1)
```

Comment:
Let's give a reason here as to why kraken exited

Reply:
Added this in the latest commit